### PR TITLE
File system connector

### PR DESCRIPTION
**Pull request by patidarpraveen244**
Allowing to read json from file system.

### CODE REVIEW

**Review by ajeybk — APPROVED** [review]

**Comment by ajeybk** [comment]
Fix the tests that are failing, then we should be able to merge.

File: docs/docs/snapshots/json.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ To setup a **JSON** snapshot configuration file, copy the following code to a fi
         "snapshots": [
             {
                 "source": "gitConnector",
```
Comment (farchide) [inline docs/docs/snapshots/json.md:23]: type should be `filesystem`

File: realm/fileSystemConnector.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+  "fileType": "structure",
+  "companyName": "prancer-test",
```
Comment (farchide) [inline realm/fileSystemConnector.json:3]: we should have "type":"filesystem" here

**Review by farchide — CHANGES_REQUESTED** [review]

File: docs/docs/snapshots/json.md (hunk 1 above)
Comment (patidarpraveen244) [reply docs/docs/snapshots/json.md:23]: I have added type filesystem.

File: realm/fileSystemConnector.json (hunk 2 above)
Comment (patidarpraveen244) [reply realm/fileSystemConnector.json:3]: Done.

**Comment by patidarpraveen244** [comment]
@farchide please take a look at the changes. Also I have created another pull request for the docs.

**Review by ajeybk — APPROVED** [review]

**Review by farchide — APPROVED** [review]

**Pull request merged**